lib/vsc/mympirun/mpi/mpi.py

```diff
@@ -151,8 +151,37 @@ def which(cmd):
     return None
 
 
-class MPI(object):
+class RunFileLoopMPI(RunFile, RunLoop):
+    """
+    RunFile specific for MPI because intel doesn't feel like fixing their shit
+    """
+    def __init__(self, cmd, **kwargs):
+        self.options = kwargs.pop('options', None)
+        self.counter = 1
+        super(RunFileLoopMPI, self).__init__(cmd, **kwargs)
+        self.readsize = -1
+        self.seen_output = False
+
+    def _loop_process_output(self, output):
+        """
+        check if process is generating any output at all; if not, warn the user after a set amount of time
+        """
+        del output
```

Comment:
because we're (also) deriving from `RunFile`, `output` should always be empty, otherwise something weird is going on...

So, add something like:

```python
if output:
    # raise hell
```